### ISSUE TITLE
Test loading/inference on all supported Python and TF versions

### ISSUE THREAD

**Issue by mikegerber** [OCR-D/ocrd_calamari#83]
Learning from qurator-spk/eynollah#87/qurator-spk/eynollah#87 I think we should test loading/test inference on all supported Python and TF versions. The worst that can happen is to learn what isn't working.

I'd do this after #61.

**Comment by mikegerber** [comment]
The best that can happen: Have an understanding of how to save a model so it can be loaded under all (supported) circumstances.

**Comment by bertsky** [comment]
Indeed. And to make it work better, as a precaution, we should try to get Calamari to [support SavedModel format](https://github.com/Calamari-OCR/calamari/issues/304#issuecomment-1048796206). In fact, the problem [has already surfaced](https://github.com/Calamari-OCR/calamari/issues/78).

**Comment by mikegerber** [comment]
@bertsky I agree (tentatively, as I have not yet made myself familiar with it, but it sure looks like the best solution)

**Comment by mikegerber** [comment]
We already do this (the testing of Python 3.6 to 3.10), I just forgot about it :-) 

- [ ] Python 3.11 (#77) needs to be reviewed
- [ ] Why is the CircleCI result not accessible/private?
- [ ] Consider scheduled tests (TensorFlow dependency tends to break, and I only notice when something new is pushed)
- [ ] Consider removing EOL Python 3.6 (or not, Ubuntu 18.04 is still used, I think)

**Comment by mikegerber** [comment]
Python 3.11 does not work currently and I don't think I'll investigate further until we can update to Calamari 2. (→ #77)

**Comment by mikegerber** [comment]
Closing as this is already done and for the other stuff we have #72.
